### ISSUE TITLE
--js only loads js files recursively

### ISSUE THREAD

**Issue by rahbari** [google/closure-compiler#3053]
compiler doesn't load mjs file when using recursive pattern:

this works:
`--js ./dir/dir/*.mjs`

this doesn't work:
`--js ./dir/**.mjs`

**Comment by lazarljubenovic** [comment]
Isn't it supposed to be `./dir/**/*.mjs`?

**Comment by rahbari** [comment]
Nope, I almost tried everything.

**Comment by brad4d** [comment]
This is roughly where the pattern matching happens
https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/CommandLineRunner.java#L2067

I see there at least one reference specifically to `'*.js'` that could be part of the problem.

There are unit tests for it in `CommandLineRunnerTest.java`

I don't think anyone on the core team has time to look into this right now, but PRs would be welcome.
Thanks.

**Comment by saurabh-ku** [comment]
I would like to work on this issue

**Comment by blickly** [comment]
Ok, great! Feel free to assign yourself if you plan to work on it.

Does bradford's comment in https://github.com/google/closure-compiler/issues/3053#issuecomment-415172479 help point you in the right direction?

**Comment by saurabh-ku** [comment]
Yep, I have found the function where the the parsing happens and the corresponding test for it. I will start working on it right away.

However I am unable to assign the issue to myself as I am not a contributor to the project yet.

**Comment by saurabh-ku** [comment]
It seems the issue has already been resolved. I did some testing with some dummy file, mimicking the directory structure provided by the issue reporter.

```
dir
└── dir
    ├── f1.js
    ├── f2.js
    ├── f3.js
    └── m1.mjs
```

running the commands 
`java -jar compiler.jar dir/**.js dir/**.mjs`
`java -jar compiler.jar dir/**.js dir/dir/*.mjs`
`java -jar compiler.jar dir/**.js dir/**/*.mjs`

all successful compile the JS files.
